index.php

```diff
@@ -19,7 +20,31 @@
             <h4 class="font-style-micro font-size-small margin-bottom-small text-align-center"><?php the_time('d F Y'); ?></h4>
             <h3 class="margin-bottom-small text-align-center"><?php the_title(); ?></h3>
 
-            <?php the_excerpt(); ?>
+    <?php
```

Comment:
You are right I cant seem to find it perhaps I got to it thru a link to one of the older html mocks. I'll do this as is then